.github/workflows/deployment.yml

```diff
@@ -17,7 +17,7 @@ env:
 jobs:
   pr:
     name: Pull Request
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
We should be able to use the latest version again since adding the python environment fix